src/Amalgam/GeneralizedDistance.h

```diff
@@ -194,6 +194,17 @@ class GeneralizedDistance
 		return s_two_over_sqrt_pi * deviation * FastExp(-term * term) - diff * std::erfc(term); //2*sigma*(e^(-1*(diff^2)/((2*simga)^2)))/sqrt(pi) - diff*erfc(diff/(2*sigma))
 	}
 
+	//surprisal in nats of each of the different distributions given the appropriate uncertainty
+	//this is equal to the nats of entropy of the distribution plus the entropy of the uncertainty
+	//in the case of Laplace, the Laplace distribution is one nat, and the mean absolute deviation is half of that,
+	//therefore the value is 1.5
+	//these values can be computed via ComputeDeviationPartLaplace(0.0, 1) for each of the corresponding methods
+	//deviations other than 1 can be used, but then the result should be divided by that deviation, yielding the same value
+	static constexpr double s_surprisal_of_laplace = 1.5;
+	static constexpr double s_surprisal_of_laplace_approx = 1.500314205;
+	static constexpr double s_surprisal_of_gaussian = 1.1283791670955126;
+	static constexpr double s_surprisal_of_gaussian_approx = 1.128615528679644;
```

Comment:
Thoughts on keeping all the statics in one place (there are a few more in this file)? I could go either way. Looks like you have them closest to where they are used right now which works but I also like to keep track of them together so that they aren't sprinkled throughout. Fine with either way.